scripts/copyResearchDocs.ts

```diff
@@ -27,14 +32,30 @@ import { copyFile, cp, mkdir, rm, writeFile } from 'fs/promises';
         'trackhar_tmp',
     ]);
 
+    const trackerNames = adapters.reduce<Record<string, Tracker['name']>>((trackers, adapter) => {
+        if (!trackers[adapter.tracker.slug]) trackers[adapter.tracker.slug] = adapter.tracker.name;
+        return trackers;
+    }, {});
+
     // Create the research directory to not break older builds
     await mkdir('trackhar_tmp/research-docs', { recursive: true });
     // Preserve the _index.md file, which contains the information of the section.
     await copyFile('content/en/research/_index.md', 'trackhar_tmp/research-docs/_index.md');
+
     await rm('trackhar_tmp/research-docs/README.md', { force: true });
     console.info('Removing old research docs…');
     await rm('content/en/research', { recursive: true, force: true });
     console.info('Copying fresh content…');
+    // Add index.md files if there are none because hugo wants those to create the pages.
+    (await readdir('trackhar_tmp/research-docs', { withFileTypes: true }))
+        .filter((dirent) => dirent.isDirectory())
+        .forEach((dir) => {
+            writeFileSync(
+                join('trackhar_tmp/research-docs', dir.name, 'index.md'),
+                `{\n"title":"${trackerNames[dir.name]}"\n}\n&nbsp;`,
+                { flag: 'wx' }
```

Comment:
> 'w': Open file for writing. The file is created (if it does not exist) or truncated (if it exists).
> 'wx': Like 'w' but fails if the path exists.

I don't understand why you're doing that.

Reply:
Upon looking at this again: Iirc, my thinking in https://github.com/tweaselORG/meta/issues/3 was that we _could_ already have an `index.md` in a `research-docs/*` folder, which would serve as a general introduction to the tracker section.

So, I do still think that this is wrong. However, I guess if we don't implement `index.md` support now, it would be nice to be alerted to this problem if we do ever add one anywhere, so having the script fail here can stay, I guess.